envoy/certificate_provider/certificate_provider_manager.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <string>
+
+#include "envoy/common/pure.h"
+#include "envoy/config/core/v3/extension.pb.h"
+#include "source/extensions/certificate_providers/certificate_provider.h"
```

Comment:
This header should live in `envoy/`

Reply:
Sorry, didn't get your point. Do you mean I need to move this header file to another location?

Reply:
got it